tests/ubuntu1804/Vagrantfile

```diff
@@ -13,15 +13,15 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  config.vm.define "hpc02-test", primary: true do |hpc02|
+  config.vm.define "hpc02-test" do |hpc02|
```

Comment:
resolves https://github.com/Quansight/qhub-onprem/issues/32

Reply:
Makes sense. Thanks for the catch.